assets/assetsData.json

```diff
@@ -0,0 +1,54 @@
+{
+  "glock":
+    {
+      "path": "glock2.png",
+      "muzzlePositionX": 100,
+      "muzzlePositionY": 54,
+      "holdingPositionX": 25,
+      "holdingPositionY": 24,
+      "width": 100,
+      "height": 64
+    },
+  "laserGun":
+    {
+      "path": "laserGun2.png",
+      "muzzlePositionX": 200,
+      "muzzlePositionY": 39,
+      "holdingPositionX": 70,
+      "holdingPositionY": 29,
+      "width": 200,
+      "height": 59
+    },
+  "laserBeam": {
+    "path": "laserBeamFrames.png",
+    "frameWidth": 1036,
+    "frameHeight": 130,
```

Comment:
I think that every sprite should be an animation. Should we add the duration in seconds of the animation ?

Reply:
You need to add some description about the fields.